### PR TITLE
Added get requests to front end and server side for related products, reviews, and outfit population. Added partial star functionality

### PR DESCRIPTION
**Pull request by TeresaGobble**

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -31,4 +31,62 @@ app.get('/products/:product_id', (req, res) => {
    })
 })
 
-// TODO: app.get('/products/reviews/:product_id') => https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfe/reviews?product_id=37312
\ No newline at end of file
+// return product's related product array at endpoint id's 'related content'
+app.get('/products/:product_id/related', (req, res) => {
```
Comment (clwilshire) [inline server/index.js:35]: does :product_id dynamically render? or do we need /products/{product_id}/related

File: client/components/Related-items-and-comparisons/Stars.jsx [hunk 2]
```diff
@@ -2,9 +2,44 @@ import React from "react";
...
+  // take total and divide by number of reviews to get average rating
+  let average = totalOfRatings/totalCount;
+
+  // rounding down to nearest 25th percentile
+  let roundedDownRating = (Math.round(average * 4) / 4).toFixed(2);
+
+
+  // There should be a total of 20 star rendering options, populated on the screen depending on the average rating calculation (each star rating image must map to the calculated value in the component)
+
+  // How to render 1/4, 1/2, 3/4 parts of star? does fontawesome have this, or must these be made and imported as assets?
```
Comment (clwilshire) [inline client/components/Related-items-and-comparisons/Stars.jsx:34]: font awesome has half stars, but I don't see quarter stars at first glance https://fontawesome.com/search?q=star&s=solid%2Cbrands

File: client/components/Related-items-and-comparisons/Stars.jsx [hunk 3]
```diff
@@ -2,9 +2,44 @@ import React from "react";
...
+  console.log(props.reviews)
+  return (
+    <>
+    <div>{props.reviews.results[0].rating}</div>
+    <div>☆☆☆☆☆</div>
+    </>
+  )
 }
 
 export default Stars;
```
Comment (clwilshire) [inline client/components/Related-items-and-comparisons/Stars.jsx:45]: we are exporting Stars, but currently not importing in any other component?

**Review by clwilshire — APPROVED** [review]
Looks great!

**Review by yellowstrings — APPROVED** [review]

**Pull request merged**